README.md

```diff
@@ -113,6 +113,26 @@ database. When prompted to select the set of initial security rules, select
 any option (e.g. "Start in Production Mode") since these permission settings
 will be overwritten below.
 
+#### Storage Setup
+
+Visit the "Storage" section of the console and create a storage bucket. In 
```

Comment:
The double quotes look good to me but it could qualify for backticks. @hsubox76 what do you think here?

Reply:
I'm happy to change it, but the rest of the file uses double quotes, so if we change one, we should probably change the rest

Reply:
in that case, double quotes is good